samcli/commands/build/command.py

```diff
@@ -101,6 +107,23 @@
     help="If your functions depend on packages that have natively compiled dependencies, use this flag "
     "to build your function inside an AWS Lambda-like Docker container",
 )
+@click.option(
+    "--container-env-vars",
+    "-e",
+    default=None,
+    multiple=True,  # Can pass in multiple env vars
+    required=False,
+    help="Input environment variables through command line to pass into build containers, you can either "
+    "input function specific format (FuncName.VarName=Value) or global format (VarName=Value). Example: "
+    "sam build --use-container --container-env-vars Func1.VAR1=value1 --container-env-vars VAR2=value2",
+)
+@click.option(
+    "--container-env-vars-file",
+    "-ef",
+    default=None,
+    type=click.Path(),  # Must be a json file
+    help="Path to environment variable json file (ex: env_vars.json) to pass into build containers",
```

Comment:
nit: AFAIK `ex` is not standard for "example"; better to use `e.g. env_vars.json`

Reply:
looks like we have some usage of `ex:` in our current codebase. 

<img width="1190" alt="image" src="https://user-images.githubusercontent.com/3804518/109231189-f2e8f180-777a-11eb-9060-0ebbd5325c7c.png">

and similar amount of `e.g.,`

<img width="1185" alt="image" src="https://user-images.githubusercontent.com/3804518/109231236-07c58500-777b-11eb-9a3f-2ff637d4c99c.png">

Reply:
In any case the standard notation for "for example" is "e.g.", see for example:
- https://english.stackexchange.com/a/16209
- https://blog.prepscholar.com/ie-vs-eg-vs-ex-definition

Reply:
I think it is worth to go extra mile to make all `ex:` into `e.g.,`

Reply:
right, I have already changed the manifest one. we can create a separate pr to change all of them